draft-ietf-rats-corim.md

```diff
@@ -771,26 +779,24 @@ The types defined for a group identified are UUID and variable-length opaque byt
 
 ##### Measurements
 
-Measurements can be of a variety of things including software, firmware,
-configuration files, read-only memory, fuses, IO ring configuration, partial
-reconfiguration regions, etc. Measurements comprise raw values, digests, or
-status information.
+A measurement can be directly compared with attestation evidence, or it can be a quality that is not directly comparable with evidence.
+The scope of a mroperty is defined by the `environment-map` and optional `mkey` it appears with.
 
-An environment has one or more measurable elements. Each element can have a
-dedicated measurement or multiple elements could be combined into a single
-measurement. Measurements can have class, instance or group scope.  This is
-typically determined by the triple's environment.
```

Comment:
I find lowercase "environment" difficult to map to a single concept. Here's my understanding. An `environment-map` does not contain measurements. An Attesting Environment has measurable elements. A Target Environment has reference values for elements.

I reword or reformat the rest of these sentences below. I don't like "typically" here since measurements always are within the scope on an environment-map.

Reply:
An environment (aka Target Environment) is the thing that is being measured by an Attesting Environment. The name of the environment is expressed as key-value pairs (which is what environment-map and stateful-environment-record do). An AE collects measurements about the TE and those measurements are expressed as key-value pairs. The measurement-value-map structure does this. 

We can change "environment" to use TE if lower case environment is confusing.
I'm OK with removing "typically".